.github/workflows/check.yml

```diff
@@ -67,6 +72,90 @@ jobs:
             rust-version:  ${{ needs.get-rust-versions.outputs.stable }}
             cache: true
 
+    check-pretty-printers:
+        needs: [ calculate-git-info, get-rust-versions ]
+        strategy:
+            # `fromJSON` is used here to convert string output to boolean
+            fail-fast: ${{ fromJSON(needs.calculate-git-info.outputs.is_bors_branch) }}
+            matrix:
+                # TODO: check on macos as well
+                os: [ ubuntu-latest, windows-latest ]
+                # `fromJSON` is used here to convert string output to sequence.
+                # `all` is a string with list of old, stable and nightly versions.
+                # Make sequence from two outputs with version is not possible here.
+                # TODO: check on more rustc versions
+                rust-version: ${{ fromJSON(needs.get-rust-versions.outputs.all) }}
+                platform-version: [ 223, 231 ]
+        runs-on: ${{ matrix.os }}
+        env:
+            ORG_GRADLE_PROJECT_platformVersion: ${{ matrix.platform-version }}
+            ORG_GRADLE_PROJECT_compileNativeCode: false
+            RUST_BACKTRACE: 1
+
+        steps:
+            - uses: actions/checkout@v3
+
+            - name: Set up Rust
+              uses: actions-rs/toolchain@v1
+              with:
+                  profile: minimal
+                  toolchain: ${{ matrix.rust-version }}
+                  default: true
+
+            - name: Set up additional env variables
+              if: matrix.rust-version == needs.get-rust-versions.outputs.old
```

Comment:
Actually, nice catch!
I drop changes with `all` versions.
We still check pretty printers with old rust version only with "old" IDE versions but don't duplicate them as with previous versions

Ideally, we want to check all possible variants but it may produce too many jobs on CI. So I think the current approach is enough. At least, for now